src/models/humidity.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+
+from src.db.db import Base
+
+
+class Humidity(Base):
+    __tablename__ = "humidity"
+
+    id = Column(Integer, primary_key=True, index=True)
+    humidity = Column(Integer)
```

Comment:
As Joe says, please change the `humidity` column type to Float.

Reply:
Done